gaps/gap-25_golem_certificates/gap-25_golem_certificates.md

```diff
@@ -0,0 +1,181 @@
+---
+gap: 25
+title: Golem Certificates structure
+description: Describes certificates usage to enable more sensitive golem features.
+author: <Witold Dzięcioł (@nieznanysprawiciel)>
+status: Draft
+type: Feature
+requires: GAP-5, GAP-4
+---
+## Abstract
+
+Some Golem features are more sensitive, because of potential malicious usage. A good example is outbound network traffic, where malicious user could cause harm, if he was able to access any internet resource. In GAP-5 and GAP-4 we decided to limit Requestor abilities by introducing Computational Manifest, which defines allowed commands and domains access. Payload Manifest is signed to confirm, that image is safe to use.
```

Comment:
```suggestion
Some Golem features are more sensitive because of potential malicious usage. A good example is outbound network traffic, where malicious user could cause harm if he was able to access any internet resource. In GAP-5 and GAP-4 we decided to limit Requestor abilities by introducing Computational Manifest which defines allowed commands and domains access. Payload Manifest is signed to confirm that image is safe to use.
```